README.md

```diff
@@ -1,2 +1,27 @@
-# The-Capstone
-Microverse CApstone Project
+# Capstone Project: Similar to real-world project.
```

Comment:
- [ ] Please include an `Acknowledgements` section in your readme file and give credit to [Cindy Shin](https://www.behance.net/adagio07) as demanded in your [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)
- [ ] Also, please include a link to the video that you recorded explaining your project. If you did not record it yet, remember to do it as this is a requirement in order to pass this capstone.